src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -137,6 +137,23 @@ function getDictionaryKey(inst) {
   return '.' + inst._rootNodeID;
 }
 
+function isInteractive(tag) {
+  return tag === 'button' || tag === 'input' ||
```

Comment:
```
return (
  tag === 'button' || tag === 'input' ||
  tag === 'select' || tag === 'textarea'
);
```

FB style doesn't do large indents to line up with previous lines – we just move the previous line down instead.